File: trunk/unityClientSample/Assets/src/protocol/AuthRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// connector上的验证请求 协议:6
    /// </summary>
	[Proto(value=6,description="connector上的验证请求")]
	[ProtoContract]
	public class AuthRequest
	{
        /// <summary>
        ///  唯一id
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public int ucenterId;
        /// <summary>
        ///  用户名
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string userCode;
        /// <summary>
        ///  是否是测试号
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public bool testing;
        /// <summary>
        ///  登陆的服务器id
        /// </summary>
		[ProtoMember(4, IsRequired = true)]
		public int serverid;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/auth/AuthRequest.java
/**
 * connector上的验证请求
 */
package org.ngame.protocol.auth;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import org.ngame.protocol.annotation.Proto;
import lombok.Data;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = 6, description = "connector上的验证请求")
public class AuthRequest
{
//  @Protobuf(required = true, description = "用户中心的加密id")
//  private String loginid;
  @Protobuf(description = "唯一id")
  private int ucenterId;
  @Protobuf(description = "用户名")
  private String userCode;
  @Protobuf(description = "是否是测试号")
  private boolean testing;
  ////////////////////////////////
  @Protobuf(required = true, description = "登陆的服务器id")
  private int serverid;
}
